components/PopupMenu/PopupMenu.js

```diff
@@ -56,6 +57,15 @@ export default function PopupMenu({
               {prop.notes ? "notes" : "+ notes"}
             </StyledMenuButton>
           </Link>
+          {prop.query2 ? null : (
```

Comment:
you don't need a ternary operator here, you could check if the opposite of your condition is true: 

```javascript
{!prop.query && (...) }
```

Reply:
yes!! I always forget about the ! 🙈